docs/reference/diagrams.md

```diff
@@ -260,3 +260,43 @@ erDiagram
 </div>
 
   [entity-relationship diagram]: https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram
+
+### Using Gitgraphs
```

Comment:
Please remove this section – we're not officially supporting this type of diagram.

Reply:
I've added a note for the other diagram types in 314bab53c.